config/channel-settings.json

```diff
@@ -35,32 +43,30 @@
                 "type": "object",
                 "properties": {
                   "bundleExpressionAlias": {
-                    "title": "Bundle Expression Alias",
-                    "type": "string",
-                    "default": null
+                    "title": "Bundle Expression Alias *",
+                    "type": "string"
                   },
                   "path": {
-                    "title": "Path",
-                    "type": "string",
-                    "default": null
+                    "title": "Path *",
+                    "type": "string"
                   },
                   "fileNameMask": {
-                    "title": "File Name Mask (Regular Expression)",
-                    "type": "string",
-                    "default": null
+                    "title": "File Name Mask (Regular Expression) *",
+                    "type": "string"
                   },
                   "documentXPath": {
                     "title": "Where in the XML is the business document? (XPath Expression)",
-                    "type": "string",
-                    "default": null
+                    "type": "string"
```

Comment:
documentXPath is not required? I don't recall if there was discussion about this. Was there reasoning for not making it required?